src/mavlink-router/ulog.cpp

```diff
@@ -191,7 +191,7 @@ int ULog::write_msg(const struct buffer *buffer)
         mavlink_msg_logging_ack_encode(LOG_ENDPOINT_SYSTEM_ID, MAV_COMP_ID_ALL, &msg, &ack);
         _send_msg(&msg, _target_system_id);
         /* message will be handled by MAVLINK_MSG_ID_LOGGING_DATA case */
-        [[gnu::fallthrough]];
+        /* fall through */
```

Comment:
?

Reply:
This removes compile warning, source:
https://stackoverflow.com/questions/45349079/how-to-use-attribute-fallthrough-correctly-in-gcc